test/functional/apps/discover/_data_grid_field_data.ts

```diff
@@ -27,7 +27,11 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
   const toasts = getService('toasts');
   const queryBar = getService('queryBar');
   const PageObjects = getPageObjects(['common', 'header', 'discover', 'visualize', 'timePicker']);
-  const defaultSettings = { defaultIndex: 'logstash-*', 'doc_table:legacy': false };
+  const defaultSettings = {
+    defaultIndex: 'logstash-*',
+    'doc_table:legacy': false,
+    'discover:searchFieldsFromSource': false,
+  };
```

Comment:
here I think setting searchFieldsFromSource to false is fine, since it's a test for a flagged feature